src/test/java/Homework18.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+
+    public void playSong() {
+      provideEmail("randy.davila@testpro.io");
```

Comment:
Hi,
I have checked your homework 18, but Test script is failed! You got this problem, please correct it.
Best wishes from the HW team (Khanh, khanh@testpro.io)

I don't see you add step navigate to koel web, you can add step `navigateToPage();` before getting into work  provideEmail